src/layer/const.ts

```diff
@@ -99,3 +100,52 @@ export const SH_SERVICE_HOSTNAMES_V3: string[] = [
   'https://services-uswest2.sentinel-hub.com/',
   'https://creodias.sentinel-hub.com/',
 ];
+
+export type GetStatsParams = {
+  fromTime: Date;
+  toTime: Date;
+  resolution: number;
+  geometry: Polygon;
+  bins?: number;
+};
+
+export type FisPayload = {
+  layer: string;
+  crs: CRS_IDS;
+  time: string;
+  resolution: number | string;
+  geometry?: string;
+  type?: HistogramType;
+  bins?: number;
+  evalscript?: string;
+  evalsource?: string;
+};
+
+export enum HistogramType {
+  EQUALFREQUENCY = 'EQUALFREQUENCY',
+  // EQUIDISTANT = 'EQUIDISTANT',
+  // STREAMING = 'STREAMING',
+}
+
+export type DailyChannelStats = {
+  date: Date;
+  basicStats: {
+    min: number;
+    max: number;
+    mean: number;
+    stDev: number;
+  };
+  histogram: {
+    bins: [
+      {
+        lowEdge: number;
+        mean: number;
+        count: number;
+      },
+    ];
+  };
+};
+
+export type GetStats = {
+  [key: string]: [DailyChannelStats];
```

Comment:
Just noticed: is this correct? IIUC, this is an array with a single element of `DailyChannelStats`? I think this is how it should be:
```
  [key: string]: DailyChannelStats[];
```

Reply:
Yes you are correct, thanks for noticing